### PR TITLE
Commit testowy

### PR DESCRIPTION
**Pull request by dracula1789**

### CODE REVIEW

File: NOTES.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+2017
+od dawna nic się nie dzieje
```
Comment (keszybz) [inline NOTES.txt]: Zdanie musi zaczynać się wielką literą.

**Comment by ccluri** [comment]
That depends! See other branches!  https://github.com/Neuroinflab/kCSD-python/tree/ipynb_tests  :D

File: NOTES.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 2017
-od dawna nic się nie dzieje
+od dawna nic się nie prawie dzieje
```
Comment (keszybz) [inline NOTES.txt]: I brakuje kropki, albo wykrzyknika!

**Comment by danek8317** [comment]
Vampires...

**Comment by keszybz** [comment]
On Wed, Dec 13, 2017 at 12:53:24PM +0000, danek8317 wrote:
> danek8317 requested changes on this pull request.
> 
> It would actually be useful to leave the comments in English ;)

I'll rework the PR.

**Pull request closed without merging**